support/generated_data.py

```diff
@@ -0,0 +1,5 @@
+from utils.fake_data_generators import *
+
+generated_title = get_fake_name()
```

Comment:
How about `getTitle`? 
in this case we would be sure that our method returns some data 
Now it is not entirely clear where and what is generated